docs/data/joy/components/accordion/accordion.md

```diff
@@ -14,10 +14,10 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/accordion/
 
 Joy UI provides four accordion-related components:
 
-- [`AccordionGroup`](#basic-usage): A container that groups multiple accordions. It **does not** control the state of each accordion.
-- [`Accordion`](#basic-usage): A component that contains the expansion logic and send to AccordionSummary and AccordionDetails.
-- [`AccordionSummary`](#basic-usage): A header of the accordion which contain a button that triggers the expansion.
-- [`AccordionDetails`](#basic-usage): A wrapper for the accordion details.
+- [Accordion Group](#basic-usage) - a container that groups multiple accordions. It **does not** control the state of each accordion.
+- [Accordion](#basic-usage) - a component that contains the expansion logic and send to AccordionSummary and AccordionDetails.
+- [Accordion Summary](#basic-usage) - a header of the accordion which contain a button that triggers the expansion.
+- [Accordion Details](#basic-usage) - a wrapper for the accordion details.
```

Comment:
Broken anchor link:

```suggestion
- [Accordion Details](#basic) - a wrapper for the accordion details.
```

But personally, this would make even more sense, it's a React component:

```suggestion
- [`AccordionDetails`](#basic) - a wrapper for the accordion details.
```

Reply:
There are other broken links like this throughout some of the Joy UI components that I'm aware of and plan to fix soon!

Reply:
+1 for the latter